src/OpenTelemetry/Sdk.cs

```diff
@@ -47,6 +48,11 @@ static Sdk()
         /// </summary>
         public static bool SuppressInstrumentation => SuppressInstrumentationScope.IsSuppressed;
 
+        /// <summary>
+        /// Gets meter to create instruments for reporting metrics on Sdk internal state.
+        /// </summary>
+        internal static Meter SdkInternalStateMeter { get; } = new Meter("SdkInternalStateMeter");
```

Comment:
Given this is still draft, could you also expose one metric using this meter, so we can get a feel of the end state? 
(The circularbuffer dropped count is a good candidate for exposing now.)

Reply:
Added